configs/packageConfig.js

```diff
@@ -1,21 +1,89 @@
+const fs = require('fs');
+const path = require('path');
+const util = require('util');
+const crypto = require('crypto');
+const readline = require('readline');
+const template = require('lodash.template');
+
 const { Platform, build } = require('electron-builder');
 
+const readFileAsync = util.promisify(fs.readFile);
+const writeFileAsync = util.promisify(fs.writeFile);
 const args = process.argv.slice(2);
 const targets = args && args.length ? args : null;
 
 if (!targets) {
   throw new Error("No arguments provided. Usage example: 'node ./packageConfig.js {mac/linux/windows}'");
 }
 
+const fileHashList = [
+  'File="<%= dmg_installer %>" | Hash=<%= dmg_sha512 %>',
+  'File="<%= exe_installer %>" | Hash=<%= exe_sha512 %>',
+  'File="<%= deb_installer %>" | Hash=<%= deb_sha512 %>',
+  'File="<%= snap_installer %>" | Hash=<%= snap_sha512 %>',
+  'File="<%= AppImage_installer %>" | Hash=<%= AppImage_sha512 %>'
+];
+
 const nodeFiles = {
   mac: { from: 'node/mac/', to: 'node/' },
   windows: { from: 'node/windows/', to: 'node/' },
   linux: { from: 'node/linux/', to: 'node/' }
 };
 
+const getFileHash = async ({ filename }) => {
+  const shasum = crypto.createHash('sha512');
+  const fileContent = await readFileAsync(filename);
+  const hashsum = shasum.update(fileContent);
+  const hash = hashsum.digest('hex');
+  return hash;
+};
+
+const generateFile = async ({ destination, lines }) => {
+  try {
+    if (!fs.existsSync(destination)) {
```

Comment:
generateFile is called once, thus file doesn't exist and this check is redundant

Reply:
ok

Reply:
generateFile will be called once per target batch.